packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -40,7 +40,7 @@ type Props = {|
     /**
      * The items present in the Dropdown
      */
-    menuItems: Array<Item> | Item,
+    menuItems: Array<Item | false | typeof undefined | null> | Item,
```

Comment:
I think this can just be `Array<?(Item | false)> | Item`.

Reply:
I spoke to at @kevinbarabash about this and he recommended including the null type too

Reply:
THe `?` modify includes allowing `null`.

Reply:
oh didn't catch the `?` there, yup that looks good to me! Thanks for the suggestion, I'll update the PR

Reply:
`Array<?Item | false>` should also work.  Also, if you ever need `typeof undefined` you should use `void` instead.